docs/DeveloperGuide.md

```diff
@@ -845,16 +857,60 @@ testers are expected to do more *exploratory* testing.
 
 ### Deleting a volunteer
 
-1. Prerequisites: List all volunteers using the `vlist` command. Multiple volunteers in the list.
+1. Deleting a volunteer while all volunteers are being shown
+
+  1. Prerequisites: List all volunteers using the `vlist` command. Multiple volunteers in the list.
+
+  1. Test case: `vdelete 1`<br>
+     Expected: First volunteer is deleted from the list. Details of the deleted volunteer shown in the status message.
+
+  1. Test case: `vdelete 0`<br>
+     Expected: No volunteer is deleted. Error details shown in the status message.
+
+  1. Other incorrect delete commands to try: `vdelete`, `vdelete x`, `...` (where x is larger than the list size or a non-integer)<br>
+     Expected: Similar to previous.
+
+### Listing all events
```

Comment:
tbh I dont think need `elist` instruction. Cos its like implicitly assumed that `vlist` and `elist` are already known to the user, for e.g. in the `deleting a volunteer` instruction the prerequisite is simply to `vlist` all volunteers so its like assumed the user alr knows about vlist without having to read an instruction about it

Reply:
u make a good point! ok ill wait until 1 more person chime in first then ill decide

Reply:
@tiongMax STUPID

Reply:
??? i respectfully disagree just in case the user performs an `efind` command and doesn't know how to go back. this is fine as it is

Reply:
but thanks max you have welcomed a dissenting opinion welcome to cs2103t debate club

Reply:
yay ok guess i wont change